authors/settings.py

```diff
@@ -189,4 +189,4 @@
 EMAIL_USE_TLS = os.getenv('EMAIL_USE_TLS')
 EMAIL_PORT = os.getenv('EMAIL_PORT')
 EMAIL_HOST_USER = os.getenv('EMAIL_HOST_USER')
-EMAIL_HOST_PASSWORD = os.getenv('HOST_PASSWORD')
\ No newline at end of file
+EMAIL_HOST_PASSWORD = os.getenv('EMAIL_HOST_PASSWORD')
```

Comment:
Great Job @jennizalwango however, why did you have to change `EMAIL_HOST_PASSWORD` environment variable?

It's throwing errors 
<img width="1182" alt="Screenshot 2019-06-11 at 08 46 20" src="https://user-images.githubusercontent.com/38750268/59246587-6bb1be00-8c25-11e9-966d-35959a6ba557.png">

Reply:
this has been cleared, it   was about the Heroku environment variable on my review app but all is fine now